app/admin/store_products.rb

```diff
@@ -1,10 +1,12 @@
 ActiveAdmin.register Product do
   menu parent: I18n.t('activeadmin.titles.stores')
 
-  permit_params :store_id, :name, :price, :link, :email, :gender, :age, :novelty, :status
+  permit_params :store_id, :name, :category_id, :price, :link,
+                :email, :gender, :age, :novelty, :status
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.

Reply:
La configuración de Rubocop que tenemos permite ambos 👀